### ISSUE TITLE
Ruby 2.5+ Compatibility

### ISSUE THREAD

**Issue by pboling** [kachick/striuct#70]
There is a deprecation warning triggered by this gem when running in Ruby 2.5, which I presume will become an error in a future version of Ruby.

Breaking into multiple lines for readability of the warning's details:
```
... /vendor/bundle/ruby/2.5.0/gems/striuct-0.6.0/lib/striuct/instancemethods/object.rb:17:
warning:
Striuct::InstanceMethods#_autonyms at 
/usr/local/rvm/rubies/ruby-2.5.1/lib/ruby/2.5.0/forwardable.rb:157
forwarding to private method Class#_autonyms
```

**Comment by kachick** [comment]
Thanks for your report! I'll work on it 💪

**Comment by kachick** [comment]
* https://github.com/ruby/ruby/commit/ef13e60fe07766549275c19f3e5112fb7773cf38
* https://bugs.ruby-lang.org/issues/12782

📝

**Comment by kachick** [comment]
@pboling Fixed and released 0.6.1 🙇

**Comment by pboling** [comment]
Thank you!!!
